RELEASE_NOTES

```diff
@@ -1,3 +1,32 @@
+========================================================================
+Amazon CloudWatch Agent 1.300030.0 (2023-10-24)
+========================================================================
+
+Bug fixes:
+* Fix EMF log corruption caused by highly concurrent messages sent to the agent
+* CA Bundle fix for AWS Secret regions
+* Filter out terminated pods when calculating container insights node request metrics to match kubectl implementation
+* Fix auto_removal to not block on the file being removed
```

Comment:
Can we also add the fix from https://github.com/aws/amazon-cloudwatch-agent/pull/915 to the release notes?